internal/controller/postgrescluster/volumes.go

```diff
@@ -449,6 +449,7 @@ func (r *Reconciler) reconcileMovePGDataDir(ctx context.Context,
 					Image:        config.PostgresContainerImage(cluster),
 					Name:         naming.ContainerJobMovePGDataDir,
 					VolumeMounts: []corev1.VolumeMount{postgres.DataVolumeMount()},
+					Resources:    cluster.Spec.InstanceSets[0].Resources,
```

Comment:
:thinking: Is there any chance `InstanceSets` might ever be empty? Perhaps a `len(cluster.Spec.InstanceSets) > 0` check above might be useful?

Reply:
:thinking: I don't think that is possible based on spec validation.. Maybe in a shutdown scenario? Either way would be good to have a check